### PR TITLE
Update/opentok - Add Broadcast and Signal api calls.

### PR DESCRIPTION
**Pull request by valencar**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [ ] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Docs: 

https://tokbox.com/developer/rest/#send_signal
https://tokbox.com/developer/rest/#start_broadcast
https://tokbox.com/developer/sdks/node/

The changes looks big because I ran the prettier. 

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [x] Include [tests for your changes](https://github.com/DefinitelyTyped/DefinitelyTyped#testing)
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

### CODE REVIEW

**Comment by CatGuardian** [comment]
Can you undo the prettier so I can review easier please?

File: types/opentok/index.d.ts [hunk 1]
```diff
@@ -77,18 +79,70 @@ declare module 'opentok' {
...
+      type: string;
+      data: any;
+    }
   }
 
   class OpenTok {
     constructor(apiKey: string, apiSecret: string);
 
-    public createSession(options: OpenTok.SessionOptions, callback: (error: Error | null, session?: OpenTok.Session) => void): void;
+    public createSession(options: OpenTok.SessionOptions, callback: (error: Error, session: OpenTok.Session) => void): void;
```
Comment (CatGuardian) [inline types/opentok/index.d.ts]: I'm not sure but I think the '| null' statements are for typescript transpiles that use the strict null checks and what not.

Comment (CatGuardian) [reply types/opentok/index.d.ts]: To be safe I would leave the nulls alone.

Comment (valencar) [reply types/opentok/index.d.ts]: Ah.. yeah, I removed for testing, I should put it back.

**Comment by CatGuardian** [comment]
Looks good. Thank you for helping out!

**Comment by CatGuardian** [comment]
Lol. Do we want the prettier or not?

I reviewed it with the prettier intact and it looked fine. So you can do prettier if you want. What do you think?

**Comment by valencar** [comment]
> Lol. Do we want the prettier or not?
> 
> I reviewed it with the prettier intact and it looked fine. So you can do prettier if you want. What do you think?

Haha! \o/ Good question. I think we were good without the prettier.

**Comment by CatGuardian** [comment]
Ok sounds good. Approved.

**Review by CatGuardian — APPROVED** [review]

**Comment by valencar** [comment]
Ready to merge

**Pull request merged**